requirements/requirements_dev.in

```diff
@@ -1,18 +1,20 @@
+Sphinx<4.4; python_version <= '3.7'
+Sphinx>4.1; python_version > '3.7'
+aiosqlite>=0.17
 black>=21.12b0
 darglint>=1.8.1
-flake8>=4.0.1
 flake8-docstrings>=1.6
 flake8-rst-docstrings>=0.2.5
+flake8>=4.0.1
 m2r2>=0.3.2
 mongomock>=3.19
-mypy>=0.910
+mongomock_motor>=0.0.2
 mypy-extensions>=0.4.3
+mypy>=0.910
 pip-tools>=6.6.2
-pytest>=6.2
 pytest-cov>=3.0
+pytest>=6.2
 recommonmark>=0.7
-Sphinx<4.4; python_version <= '3.7'
-Sphinx>4.1; python_version > '3.7'
 sphinx-rtd-theme>=0.5
-versioneer>=0.21
-twine>=3.8
\ No newline at end of file
+twine>=3.8
+versioneer>=0.21
```

Comment:
This is actually done according to PyCharm sort lines capabilities

I just ran PyCharm sorting . 

If this is the default behavior of IDEs , I would probably keep it like this since it will be easier to maintain